apps/backend/db_patches/db_seeds/0005_Questions.sql

```diff
@@ -370,7 +370,7 @@ INSERT INTO templates_has_questions(
 VALUES 
   (
     'dynamic_multiple_choice_question', 
-    1, 5, 6, '{"variant":"dropdown","options":["One","Two","Three"],"isMultipleSelect":true}'
+    1, 5, 6, '{"variant":"dropdown", "url":"", "jsonPath":"","isMultipleSelect":true, "apiCallRequestHeaders":[]}'
```

Comment:
This patch will not run on an already existing db, we should create a new db patch

Reply:
If I'm not mistaken, all the patches in the db_seeds folder serve E2E test only